invenio_drafts_resources/services/records/tasks.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-Drafts-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Celeris tasks to manage drafts."""
+
+from datetime import datetime, timedelta
+
+from celery import shared_task
+from invenio_base.utils import obj_or_import_string
+from invenio_db import db
+
+
+@shared_task(ignore_result=True)
+def cleanup_drafts(current_service_imp, seconds_param=3600):
```

Comment:
moderate: same issue with naming, I think is better without `param` (but is a personal opinion)